lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -54,7 +51,7 @@ class FeatureCard extends React.Component {
   }
 
   getCampaigns(staffPickOnly, paginate, page, callback) {
-    this.requests.push(this.serverRequest = $.get(`${API_URL}campaigns?${staffPickOnly ? 'staff_pick=true&' : ''}${page != -1 ? 'page=' + page : ''}`, result => {
+    this.requests.push(this.serverRequest = $.get(`${this.apiClient.url}campaigns?${staffPickOnly ? 'staff_pick=true&' : ''}${page != -1 ? 'page=' + page : ''}`, result => {
```

Comment:
Yes, and I tested the calls with simpler stuff via the `FeatureCard` but wanted to do the full switcheroo in a different PR to minimize the testing and changes. Upcoming PR will have those changed. Mostly setting the groundwork for the `ApiClient` class and defs used in `ReportbackItemKudos` component... and slowly extending it out elsewhere ;)